src/features/mail/Mail.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react";
+import { Modal } from "react-bootstrap";
+
+import { Inbox } from "./components/Inbox";
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+
+import { Context } from "./MailProvider";
+
+import baldMan from "../../assets/npcs/bald_man.png";
```

Comment:
We have aliased these directories in our tsconfig.

Consider using `assets/npcs/bald_man.png`. This makes it easier if we change the directory structure as well